packages/strapi-design-system/src/Pagination/components.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+/** TODO: make sure to use the react component when they are available in strapi-icons */
+import prevLink from './assets/previous.svg';
+import nextLink from './assets/next.svg';
+import { VisuallyHidden } from '../VisuallyHidden';
+import { useActivePage } from './PaginationContext';
+import { Text } from '../Text';
+
+const Wrapper = styled.a`
```

Comment:
Are you sure that using an `a` tag will cause the browser to refresh when navigating from one page to another?

Reply:
Yes :).

But it might be a problem if we only want partial rendering of the page in a router (like react-router or reach-router)

Reply:
Do you think we should use a `<button>` instead?

Reply:
Since the pagination is a nav, I think buttons are not suited (semantically) for navigation

Reply:
You might be right, and what would be the difference with using the following?

```
<nav arial-label="pagination navigation">
<ul>
 <li> <button aria-label="Go to page X">X</button></li>
</ul>
</nav>
```

Reply:
buttons are for actions, not for navigation. https://marcysutton.com/links-vs-buttons-in-modern-web-applications

The problem is the same as using a <p> instead of a <h1>: it's the its purpose

Reply:
I think that we should anticipate the integration with our router, either in this PR or when we start migrating the admin panel to the new DS. I know that MUI for instance offers some [router integration](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md) do you think that we should also provide such support on our side?

Reply:
I think we can start with a simple anchor and add a TODO flag. Then, add a peer dependency to the router we use and smoothly move to the anchor of our routers. That way, we still have time to choose a valid and accessible router without breaking the speed of the DS implem 😊 . Does it sound fair to you?